data/sql/updates/pending_db_world/rev_1611865917828980400.sql

```diff
@@ -1,3 +1,6 @@
 INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611865917828980400');
-INSERT INTO item_template_locale (ID, locale, Name, Description, VerifiedBuild) VALUES ('30622', 'zhCN', '焰铸钥匙', '允许你进入英雄难度的地狱火堡垒地下城。', '15050'); 
-INSERT INTO item_template_locale (ID, locale, Name, Description, VerifiedBuild) VALUES ('30637', 'zhCN', '焰铸钥匙', '允许你进入英雄难度的地狱火堡垒地下城。', '15050');
\ No newline at end of file
+DELETE FROM `item_template_locale` WHERE `ID` = 30622 and locale = 'zhCN'
+DELETE FROM `item_template_locale` WHERE `ID` = 30637 and locale = 'zhCN'
```

Comment:
can be just one `DELETE`:

```sql
DELETE FROM `item_template_locale` WHERE locale = 'zhCN' AND `ID` IN (30622, 30637);
```

the rest looks good, well done!

Reply:
this was fixed several commits ago xD

Reply:
should be done already in the last commit. and thanks :)